.github/workflows/ci.yml

```
@@ -145,3 +145,33 @@ jobs:
           source pytest/venv/bin/activate
           cd pytest 
           pytest -m "not ci_excluded" -s -x
+
+  tee-launcher-tests:
+    name: "Run TEE Launcher pytests"
+    runs-on: warp-ubuntu-2204-x64-8x
+    timeout-minutes: 60
+    permissions:
+      contents: read
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+
+      - name: Setup virtualenv
+        run: |
+          python3 -m venv pytest/venv
+          source pytest/venv/bin/activate
```

just to be a bit more explicit, I would prefer if we didn't store this venv in the pytest folder for now (even though it shouldn't make a difference here).
```suggestion
          python3 -m venv tee_launcher/venv
          source tee_launcher/venv/bin/activate
```